src/main/java/io/reactivex/internal/operators/maybe/MaybeFromCallable.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
+ * the License for the specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivex.internal.operators.maybe;
+
+import java.util.concurrent.Callable;
+
+import io.reactivex.*;
+import io.reactivex.internal.disposables.EmptyDisposable;
+
+public final class MaybeFromCallable<T> extends Maybe<T> {
+
+    final Callable<? extends T> callable;
+    
+    public MaybeFromCallable(Callable<? extends T> callable) {
+        this.callable = callable;
+    }
+
+    @Override
+    protected void subscribeActual(MaybeObserver<? super T> s) {
+
+        s.onSubscribe(EmptyDisposable.INSTANCE);
```

Comment:
Setting an `EmptyDisposable` should be followed either by an `onError()` or `onComplete()` because a fused-consumer won't expect and won't request a value from it.